src/app/views/sidebar/sample-queries/sample-query-utils.ts

```diff
@@ -0,0 +1,11 @@
+export function isJsonString(str: string) {
```

Comment:
Since `JSON.parse` throws an error when called with invalid json string, this can be simplified to

```javascript
export function isJsonString(str): boolean {
  try {
    JSON.parse(str)
    return true
  } catch (error) {
    return false
  }
}
```